app/assets/stylesheets/detailsPage.scss

```diff
@@ -5,36 +5,59 @@
   max-width: 1500px;
   display: grid;
   grid-column: 2/3;
-  padding: 5px;
+  margin: 0px;
+  padding: 0px;
 }
 
 #detailsPageHeader {
   display: block;
-}
-
-#detailsPageInfo {
+  background-color: $gray-image-color;
+  position: relative;
+  width: 100vw;
+  height: 45vh;
   justify-content: center;
   text-align: center;
-  margin-left:auto;
-  margin-right:auto;
+  margin-left:0px;
+  margin-right:0px;
+  word-wrap: break-word;
 }
 
-h1 {
-  background-color: Transparent;
-  color: #000;
+.item_name {
+  color: white;
   font-weight: normal;
-  font-size: 35pt;
+  font-size: 5vw;
   font-family: 'Arvo', sans-serif;
+  background-color: Transparent;
```

Comment:
Is this necessary? I thought background-color is set to transparent by default.

Reply:
Nope; you're right. Bad habit on my part--there's a limit to how much spelling out is needed

Reply:
Sorry I forgot to mention this, but I think the same issue also occurs in a couple of other places in the code. Is it possible to fix those as well?

Reply:
It's possible, but it's not really necessary. It's usually always a good idea to spell out exactly what you want so you're not surprised. It's obviously not necessary to always spell it out, but it's not truly a bad habit, per se. Is there a limit to how much it's truly needed? Yes. Does that make it a bad idea? Not really. 

Personally, I would leave it.